{"child_modules": [], "data_mtime": 1573917617, "dep_lines": [1, 3, 4, 5, 6, 7, 1, 1, 1], "dep_prios": [5, 5, 5, 5, 5, 5, 5, 30, 30], "dependencies": ["typing", "pydantic", "pydantic.error_wrappers", "starlette.exceptions", "starlette.requests", "starlette.websockets", "builtins", "abc", "starlette"], "hash": "979b1e825dc90d3b64cf13354f04f166", "id": "fastapi.exceptions", "ignore_all": true, "interface_hash": "c65d6523dab5436e12910ce4af1f83f4", "mtime": 1573917361, "options": {"allow_redefinition": false, "allow_untyped_globals": false, "always_false": [], "always_true": [], "bazel": false, "check_untyped_defs": false, "disallow_any_decorated": false, "disallow_any_explicit": false, "disallow_any_expr": false, "disallow_any_generics": false, "disallow_any_unimported": false, "disallow_incomplete_defs": false, "disallow_subclassing_any": false, "disallow_untyped_calls": false, "disallow_untyped_decorators": false, "disallow_untyped_defs": false, "follow_imports": "silent", "follow_imports_for_stubs": false, "ignore_errors": false, "ignore_missing_imports": true, "implicit_reexport": true, "local_partial_types": false, "mypyc": false, "no_implicit_optional": false, "platform": "win32", "plugins": [], "show_none_errors": true, "strict_equality": false, "strict_optional": true, "strict_optional_whitelist": null, "warn_no_return": true, "warn_return_any": false, "warn_unreachable": false, "warn_unused_ignores": false}, "path": "C:\\Users\\Vadim\\AppData\\Local\\Programs\\Python\\Python37-32\\lib\\site-packages\\fastapi\\exceptions.py", "size": 836, "suppressed": [], "version_id": "0.740"}